### DISCUSSION TITLE
Vector search: Better to create one large entry or multiple smaller ones?

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#678] [Q&A]
So our situation is that we have alot of products and each of the products has Q&A. We want to put the Q&A into Orama, but use the vector search for this, as q&a is more semantic vs keyword. My question is, is it better to dump each question and answer as a separate document in the index or is it better to combine and put all the q&a for a specific product into one large document, and then store the q&a for each product as a document (so far fewer documents, but each document is longer because it has all the q&a for that specific product). Which approach is better for Vector search, or does it even matter? THanks.

**Comment by micheleriva** [comment]
Hi @osseonews, I think it's worth sharing some examples before making conclusions. Would you mind joining at https://orama.to/slack and send me some example data? I'd be glad to help you with this.
